.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+We are always happy to receive pull requests.
+
+Please make sure you have read our coding guidelines: https://github.com/mlr-org/mlr/wiki/mlr-Coding-Guidelines
```

Comment:
Maybe a short bullet point list of the most important points, in particular the points that are important for people doing only small fixes and unwilling to read through all of that.